en, zh: add release notes for v1.3.1 (#1694)
pingcap/docs-tidb-operator: en/releases/release-1.3.0-beta.1.md

    1. Use the `baseImage` and `version` fields to replace the `image` field. For details, refer to [Configure TiDB deployment](../configure-a-tidb-cluster.md#version).
    2. Upgrade TiDB Operator.

> - Due to the issue [#4434](https://github.com/pingcap/tidb-operator/pull/4434), If you upgrade TiFlash to v5.4.0 or later when using v1.3.0-beta.1 TiDB Operator, TiFlash maybe lose metadata and not work. If TiFlash is deployed in your cluster, it's recommended that upgrade TiDB Operator to v1.3.1 or later to upgrade TiFlash.

Reviewer: ```suggestion
- Due to the issue in [#4434](https://github.com/pingcap/tidb-operator/pull/4434), if you upgrade TiFlash to v5.4.0 or later when using v1.3.0-beta.1 TiDB Operator, TiFlash might lose metadata and not work. If TiFlash is deployed in your cluster, it is recommended that you upgrade TiDB Operator to v1.3.1 or later versions before upgrading TiFlash.
```

> - Due to the issue [#4435](https://github.com/pingcap/tidb-operator/pull/4435), when using v1.3.0 TiDB Operator, you have to set `tmp_path` field in TiFlash's config to use v5.4.0 or later TiFlash. It's recommended that upgrade TiDB Operator to v1.3.1 or later to use TiFlash.

Reviewer: ```suggestion
- Due to the issue in [#4435](https://github.com/pingcap/tidb-operator/pull/4435), when using TiDB Operator v1.3.0-beta.1, you must set the `tmp_path` field in TiFlash's config to use TiFlash v5.4.0 or later versions. It is recommended that you upgrade TiDB Operator to v1.3.1 or later versions before you deploy TiFlash.
```
